tools/cellprofiler/cellprofiler.xml

```diff
@@ -20,17 +20,34 @@
         <configfile name="script_file">
 mkdir ./input
 mkdir ./output
-#for $i, $filename in enumerate($input_images):
-    ln -s '$filename' './input/${filename.element_identifier}.${filename.ext}'
-#end for
+
+#if $con_input_type.input_type =="yes"
+    tar -C ./input -xvf $con_input_type.input_images
```

Comment:
we can keep this as an option, but might not be needed with the multi-page approach

Reply:
we decided to keep it, so no changes needed here :)